doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -583,3 +587,61 @@ Data per item is:
 * `6` : Target IPv6 address
 * `S` : RLOC16 of target
 * `C` : Age (order of use, 0 indicates most recently used entry)
+
+### PROP 5412: SPINEL_PROP_THREAD_MGMT_GET_ACTIVE_DATASET (#prop-thread-mgmt-get-active-dataset)
```

Comment:
Here should be `PROP 5413`

Reply:
Indeed. Thanks. Will rebase on master and fix.